k8s/eks-setup.md

```diff
@@ -0,0 +1,230 @@
+
+If you do not already have a bucket you want to use in the same region as your EKS cluster, create it with `aws s3 mb`, e.g.
+```
+aws s3 mb s3://zqd-demo-1
+```
+Now use zar to create an object in S3. 
+
+For the following zar import, change the directory name to match your s3 bucket.
+```
+zar import -R s3://zqd-demo-1/mark/sample-http-zng zq-sample-data/zng/http.zng.gz
+```
+This creates zng files in an s3 directory called `sample-http-zng` that we will use from zapi after zqd has been deployed. To check what zar created:
+```
+aws s3 ls zqd-demo-1/mark --recursive
+```
+
+## Deploy the zqd service with Helm
+Here is an example Helm install. This assumes you created an S3 resident space as in the local install above. This is similar to the local Helm deploy. It uses the same charts, but command line parameters overide the Values.yaml to provide specific configuration for EKS.
+
+Substitute the image.repository you created above. Note that unlike the local deployment, we do not use K8s secrets for AWS credentials because the `cluster.yaml` above specified IAM policies for S3 access.
+
+```
+helm install zqd-test-1 charts/zqd \
+  --set AWSRegion="us-east-2" \
+  --set datauri="s3://zqd-demo/mark/zqd-meta" \
+  --set image.repository="123456789012.dkr.ecr.us-east-2.amazonaws.com/" \
+  --set useCredSecret=false
+```
+
+## Exposing the zqd endpoint
+
+To access the running zqd instance, use kubectl port-forward:
+```
+kubectl port-forward svc/zqd-test-1 9867:9867 &
+```
+There is a script `k8s/zqd-port.sh` that does this after removing any existing port-forwards.
+
+## Using a K8s hosted zqd to search large logs in the cloud
+This decribes a set of steps to use `zar import` to process large log files that can then be searched with the zqd you just deployed. In order to do this, you will need an EC2 instance in the same region as your EKS cluster. Follow the steps below to set this up.
+
+## Setting up an EC2 instance to run zar import
+
+Create an EC2 instance in the same region as the S3 buckets you want to access. SSH into the instance, and:
+```
+# provide your credentials
+aws configure
+# make sure you can see the S3 buckets
+aws s3 ls
+# install golang tools from https://golang.org/dl/
+wget https://golang.org/dl/go1.14.7.linux-amd64.tar.gz
+sudo tar -C /usr/local -xzf go1.14.7.linux-amd64.tar.gz
+echo "export PATH=$PATH:/usr/local/go/bin" >> ~/.bash_profile
+source ~/.bash_profile
+go version  # make sure go is there!
+# install git, clone and install zq
+sudo yum install git -y
+git clone https://github.com/brimsec/zq
+cd zq
+make install
+~/go/bin/zar help  # make sure zar is built!
+```
+Now you have an environment where you can run zar and it can operate on data in S3 with good bandwidth. You can update it as needed with:
+```
+cd zq
+git pull
+make install
+```
+You may want to use an SSD for $TMPDIR for zar import where the sort can spill to disk. Consider this if zar runs out of space for sorting.
```

Comment:
Note: Once #1397 is merged, zar import will no longer use spill to disk.

Reply:
I am removing these lines